### PR TITLE
Improve 404 page

### PR DESCRIPTION
**Pull request by dawran6**
Add explicit 404 message on not-found page:
image

### CODE REVIEW

File: src/cljdoc/render/error.clj [hunk 1]
```diff
@@ -7,7 +7,8 @@
 (defn not-found-404 []
   (->> [:div.pt4
         [:div.mt5-ns.mt6-l.mw7.center.pa4.pa0-l
-         [:p.f2-ns.f3.mv3.w-90-l.lh-copy "We couldn't find anything here..."]
+         [:h1.f2.f1-ns.helvetica.b ":("]
```
Comment (martinklepsch) [inline src/cljdoc/render/error.clj]: Why the `.helvetica` here? cljdoc usually uses `.sans-serif` which defaults to the OS' default sans serif font.

Comment (dawran6) [reply src/cljdoc/render/error.clj]: I don't have specific reason. Just wanted to play around with different fonts to make it look a bit nicer. Let me put it back.

Comment (martinklepsch) [reply src/cljdoc/render/error.clj]: Ok cool, was just wondering 🙂

**Comment by martinklepsch** [comment]
Merged, will deploy later today 👍

**Comment by dawran6** [comment]
That's awesome. Thanks!

**Pull request merged**